lib/rubygems/core_ext/kernel_warn.rb

```diff
@@ -3,23 +3,15 @@
 # `uplevel` keyword argument of Kernel#warn is available since ruby 2.5.
 if RUBY_VERSION >= "2.5"
 
-  module Kernel
-    rubygems_path = "#{__dir__}/" # Frames to be skipped start with this path.
+  module KernelExt
```

Comment:
This declares a global `KernelExt`, should probably be under `Gem::` to avoid conflicts.